packages/synapse-react-client/src/components/ConfirmationDialog/ConfirmationDialog.tsx

```diff
@@ -51,7 +51,7 @@ export type ConfirmationDialogProps = DialogBaseProps & ConfirmationButtonsProps
 /**
  * A confirmation dialog built using MUI components.
  */
-export const ConfirmationDialog = (props: ConfirmationDialogProps) => {
+export function ConfirmationDialog(props: ConfirmationDialogProps) {
```

Comment:
Fixes a runtime error unrelated to this PR that I kept getting after pulling main - something like "attempting to use ConfirmationDialog before it is initialized"